api/location.ts

```diff
@@ -14,6 +20,51 @@ export const getLocationTags = (urlArg: URL) => async () => {
     return handleResponse<LocationTag[]>(response);
 };
 
+export const getLocationCategories = (urlArg: URL) => async () => {
```

Comment:
functions in `api` are 1-to-1 mapping of backend APIs. Move this function to `sdk.Client` as a method.

Reply:
directly call `this.getLocationTags` in it.